src/playground/sample_data_tooling/requirements.txt

```diff
@@ -11,7 +11,6 @@ faker-vehicle==0.2.0
 faker-web==0.3.1
 faker-wifi-essid==0.4.1
 faker_airtravel==0.4
-faker_education==1.2
```

Comment:
Use txt file to control the package, think about if there any other possible solution to control packages for python. this is not blocker, just leave the comment here.

Reply:
Yeah I've changed the versioning from `==` to `>=`. I know for a fact that these exact versions will work with me but I want to avoid dependency issues down the road.